tests/test_deepgrow_transforms.py

```diff
@@ -73,42 +79,42 @@
 }
 
 DATA_5 = {
-    "image": np.arange(25).reshape((1, 5, 5)),
-    "image_meta_dict": {"spatial_shape": [5, 5, 1]},
+    CommonKeys.IMAGE: np.arange(25).reshape((1, 5, 5)),
+    f"{CommonKeys.IMAGE}_meta_dict": {"spatial_shape": [5, 5, 1]},
     "foreground": [[2, 2, 0]],
     "background": [],
 }
 
 DATA_6 = {
-    "image": np.arange(25).reshape((1, 5, 5)),
-    "image_meta_dict": {"spatial_shape": [5, 2, 1]},
+    CommonKeys.IMAGE: np.arange(25).reshape((1, 5, 5)),
+    f"{CommonKeys.IMAGE}_meta_dict": {"spatial_shape": [5, 2, 1]},
     "foreground": [[2, 1, 0]],
     "background": [[1, 0, 0]],
 }
 
 DATA_7 = {
-    "image": np.arange(500).reshape((5, 10, 10)),
-    "image_meta_dict": {"spatial_shape": [20, 20, 10]},
+    CommonKeys.IMAGE: np.arange(500).reshape((5, 10, 10)),
+    f"{CommonKeys.IMAGE}_meta_dict": {"spatial_shape": [20, 20, 10]},
     "foreground": [[10, 14, 6], [10, 14, 8]],
     "background": [[10, 16, 8]],
     "slice": 6,
 }
 
 DATA_8 = {
-    "image": np.arange(500).reshape((1, 5, 10, 10)),
-    "image_meta_dict": {"spatial_shape": [20, 20, 10]},
+    CommonKeys.IMAGE: np.arange(500).reshape((1, 5, 10, 10)),
+    f"{CommonKeys.IMAGE}_meta_dict": {"spatial_shape": [20, 20, 10]},
     "guidance": [[[3, 5, 7], [4, 5, 7]], [[4, 5, 8]]],
 }
 
 DATA_9 = {
-    "image": np.arange(1000).reshape((1, 5, 10, 20)),
-    "image_meta_dict": {"foreground_cropped_shape": (1, 10, 20, 40)},
+    CommonKeys.IMAGE: np.arange(1000).reshape((1, 5, 10, 20)),
+    f"{CommonKeys.IMAGE}_meta_dict": {"foreground_cropped_shape": (1, 10, 20, 40)},
     "guidance": [[[6, 10, 14], [8, 10, 14]], [[8, 10, 16]]],
 }
 
 DATA_10 = {
-    "image": np.arange(9).reshape((1, 1, 3, 3)),
-    "image_meta_dict": {
+    CommonKeys.IMAGE: np.arange(9).reshape((1, 1, 3, 3)),
+    f"{CommonKeys.IMAGE}_meta_dict": {
```

Comment:
The "_meta_dict" part is now like a magic string, should we have that stored in `CommonKeys` and have `f"{CommonKeys.Image}_{CommonKeys.MetaDict}"`?

Reply:
see https://github.com/Project-MONAI/MONAI/pull/3671.

Reply:
Yeah that looks good though we should commit this PR first.